### ISSUE TITLE
Add i18n for others languages support

### ISSUE THREAD

**Issue by finkiki** [mariusor/go-littr#40]
Littr looks promising but it's actually only available in english. It would be nice for instances owners to deploy littr in their own languages to build their own communities. 

What do you think ?

**Comment by mariusor** [comment]
Yes, I agree, but it isn't a top priority for me, unfortunately. 

Also, I don't know any of the i18n packages for Go, so I can't really estimate the amount of work is needed for such a change.

However I would gladly accept and help out with a PR for this if you're willing to give it a try. 

We would need a first discussion about which packages for i18n we can use, and them make a choice.
After, we can establish a scope for the translations: in my opinion a first PR would do with changing just the template strings.

Let me know if you're interested, and thank you for your interest. :)

**Comment by finkiki** [comment]
I don't know anything about go packagés but can't we just directly translate template strings ?

**Comment by mariusor** [comment]
Well, I'm sorry you're not able to help with the code. Currently I don't have the time to invest in this, but I'll add it to my list of things to do before v1.0.

If you're willing, I can ping you when I have something for this for help with the translation. Which language would you be able to help with?

**Comment by finkiki** [comment]
Ok, ping me when you need my help. I can only help with french.
